app/ai/voice/agents/breeze_buddy/agent.py

```
@@ -486,8 +486,81 @@ async def run(self, runner_args: Optional[RunnerArguments] = None):
         # Only add tracing parameters when tracing is enabled
         if ENABLE_BREEZE_BUDDY_TRACING:
             setup_tracing("breeze-buddy")
-            task_params["conversation_id"] = conversation_id
-            task_params["enable_tracing"] = True
+
+            use_custom_tracing = await ENABLE_CUSTOM_TRACING()
```

Ruff (0.14.10)

Undefined name `ENABLE_CUSTOM_TRACING`

(F821)